src/components/Nav.jsx

```diff
@@ -31,7 +33,7 @@ export default function Nav() {
           <span className="badge_wrapper">
             <i className="fas fa-shopping-cart fa-2x"></i>
             <span className="badge badge_circle badge_primary position_absolute right-0 top-0">
-              4
+              {cart.length}
```

Comment:
Suggestion you can do a conditional rendering so that badge is not shown when cart is empty

Reply:
Done